packages/main/src/services/WebviewService.ts

```diff
@@ -63,7 +63,7 @@ export default class WebviewService {
       },
     });
     this.view.setBackgroundColor("#fff");
-    this.view.webContents.setAudioMuted(true);
+    this.view.webContents.setAudioMuted(false);
```

Comment:
Can you add an option in the Settings to control the sound playing of the page, the default is false

Reply:
Doable. Can you tell me why it should be muted by default? It seems strange to me.